src/Features/LanguageServer/Protocol/Handler/Diagnostics/AbstractPullDiagnosticHandler.cs

```diff
@@ -376,13 +376,13 @@ static LSP.Range GetRange(DiagnosticDataLocation dataLocation)
                 {
                     Start = new Position
                     {
-                        Character = dataLocation.OriginalStartColumn,
-                        Line = dataLocation.OriginalStartLine,
+                        Character = dataLocation.OriginalFileSpan!.Value.StartLinePosition.Character,
+                        Line = dataLocation.OriginalFileSpan!.Value.StartLinePosition.Line,
                     },
                     End = new Position
                     {
-                        Character = dataLocation.OriginalEndColumn,
-                        Line = dataLocation.OriginalEndLine,
+                        Character = dataLocation.OriginalFileSpan!.Value.EndLinePosition.Character,
+                        Line = dataLocation.OriginalFileSpan!.Value.EndLinePosition.Line,
```

Comment:
note: unclear waht semantics we want in LSP if OriginalFileSpan is null.  However, the previous code always assumed these values were here, so i'm preserving that.